Cargo.toml

```diff
@@ -42,6 +43,10 @@ members = [
     "modules/fedimint-ln-client",
     "modules/fedimint-ln-server",
     "modules/fedimint-ln-tests",
+    "modules/fedimint-ln-common-ng",
```

Comment:
Note: `ng` does not get old well. `lnv2` maybe? So in two years we don't need to have `ln-ng-ng` or something. ;)

Reply:
It's funny when you decommission hardware that was once named "NG", I've seen that a few times :laughing:

Reply:
+1 to `lnv2`

Reply:
So just to be clear, which option exactly?
fedimint-lnv2-server
fedimint-ln-v2-server
fedimint-ln-server-v2

I would assume the first as its the most consistent?

Reply:
First seems good.